### ISSUE TITLE
Doestn't work with Vue JS routing

### ISSUE THREAD

**Issue by SergiOnGit** [fluid-player/fluid-player#249]

**Comment by cian-exads** [comment]
Hi @SergiOnGit, can you provide a link to a test page where I can see the issue?

**Comment by SergiOnGit** [comment]
Sorry, i gave up already. I have nuxt js project and put fluidplayer there. It worked on page refresh, but not when i go to other route and then return back, For example: home->videos->video->otherPage->videos->video i hope you understand what i mean. I had init code in vue mounted

**Comment by kieusonlam** [comment]
I'm using Vue.js too, and I also want to try this project.

It would be greate if we have a package on `npm` that we can use it something look like this

```js
import fluidPlayer from 'fluid-player'

fluidPlayer(element)
```

**Comment by Addvilz** [comment]
Related to #119 and #430. Once this is published, you will bind `beforeDestroy` in Vue to call `player.destroy` to properly dispose of the Fluid Player instance. Haven't tested this with Vue, but you might also need to bind some navigation guards for router.

**Comment by Addvilz** [comment]
Fixed with v2.4.11
